platform/public/index.html

```diff
@@ -16,6 +16,12 @@
 <body class="h-100" onresize="fit()" onload="fit();updateGutterVisibility();">
     <div id="preloader"><img src="images/preloader.gif" width="100px"></div>
 
+    <div id='feedback-button'>
+        <p><a href="FEEDBAK_SURVEY_URL" target="_blank" rel="noopener noreferrer">
```

Comment:
`FEEDBAK` -> `FEEDBACK` 😄 

More importantly, though: how does this get instantiated effectively? I would like to ensure we don't create more and more places requiring manual adjustment on deployment. Can we automate filling this out on deployment, e.g. via a `.env` file?

Reply:
Corrected in f2298d7605d464f87af539998b569d29fd45ad73 and now settable via en environment variable. The feedback button is now only displayed if the env variable is set.